dll/win32/browseui/addressband.cpp

```diff
@@ -259,8 +259,64 @@ HRESULT STDMETHODCALLTYPE CAddressBand::HasFocusIO()
     return S_FALSE;
 }
 
+static WCHAR GetAccessKeyFromText(WCHAR chAccess, LPCWSTR pszText)
+{
+    for (const WCHAR *pch = pszText; *pch != UNICODE_NULL; ++pch)
+    {
+        if (*pch == L'&' && pch[1] == L'&')
+        {
+            /* Skip the first '&', the second is skipped by the for-loop */
+            ++pch;
+            continue;
+        }
+        if (*pch == L'&')
+        {
+            ++pch;
+            chAccess = *pch;
+            break;
+        }
+    }
+
+    ::CharUpperBuffW(&chAccess, 1);
+    return chAccess;
+}
+
+static WCHAR GetAddressBarAccessKey(WCHAR chAccess)
+{
+    static WCHAR s_chCache = 0;
+    if (s_chCache)
+        return s_chCache;
```

Comment:
This no request for change, just a thought: If we would ever implement anything like changing the active ros language at runtime, then such a caching will cause trouble if we switch between 2 languages which do use a different accelerator here. No need to react, but please do not resolve/collapse my comment!